5-AccessControl/1-call-api-roles/README.md

```diff
@@ -42,193 +59,181 @@ In the sample, a **dashboard** component allows signed-in users to see the tasks
+    1. Select **Set** next to the **Application ID URI** to generate a URI that is unique for this app.
+    1. For this sample, accept the proposed Application ID URI (`api://{clientId}`) by selecting **Save**. Read more about Application ID URI at [Validation differences by supported account types \(signInAudience\)](https://docs.microsoft.com/azure/active-directory/develop/supported-accounts-validation).
+
+##### Publish Delegated Permissions
+
+1. All APIs must publish a minimum of one [scope](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow#request-an-authorization-code), also called [Delegated Permission](https://docs.microsoft.com/azure/active-directory/develop/v2-permissions-and-consent#permission-types), for the client's to obtain an access token for a *user* successfully. To publish a scope, follow these steps:
+1. Select **Add a scope** button open the **Add a scope** screen and Enter the values as indicated below:
+    1. For **Scope name**, use `Todolist.Read`.
+    1. Select **Admins and users** options for **Who can consent?**.
+    1. For **Admin consent display name** type in the details, `e.g. Allow the users of the app msal-react-spa to read ToDo list items`.
+    1. For **Admin consent description** type in the details `e.g. Allows the app msal-react-spa to read the signed-in users ToDo list items.`
+    1. For **User consent display name** type in the details `e.g. Read ToDo list items as yourself`.
+    1. For **User consent description** type in the details `e.g. Allow the app msal-react-spa to read ToDo list items on your behalf.`
+    1. Keep **State** as **Enabled**.
+    1. Select the **Add scope** button on the bottom to save this scope.
+    > Repeat the steps above for another scope named **Todolist.ReadWrite**
+1. Select the **Manifest** blade on the left.
+    1. Set `accessTokenAcceptedVersion` property to **2**.
+    1. Select on **Save**.
+
+##### Add Scopes to msal-react-spa
+
+1. Since this app signs-in users, we will now proceed to select **delegated permissions**, which is is required by apps signing-in users.
+1. In the app's registration screen, select the **API permissions** blade in the left to open the page where we add access to the APIs that your application needs:
+    1. Select the **Add a permission** button and then,
+    1. Ensure that the **Microsoft APIs** tab is selected.
+    1. In the list of APIs, select the API `service`.
```

Comment:
>service [](http://example.com/codeflow?start=44&length=7)

there is no "service" in this sample, so check what the app reg screen shows and update accordingly